cypress/e2e/models/migration/task-manager/task-manager.ts

```diff
@@ -77,4 +82,16 @@ export class TaskManager {
         click(searchButton);
         cy.wait(2 * SEC);
     }
+
+    public static enablePreemption(rowName: string): void {
```

Comment:
I suggest to modify to a general func for enable and disable preemption

Reply:
How do you disable preemption ? There isn't any option in UI .

Reply:
Once you enable preemption on a task - the action in the kebab menu changes to "disable preemption" and you can select it as long as the task is not running